managed/services/telemetry/config.default.yml

```diff
@@ -544,3 +531,25 @@ telemetry:
     data:
       - metric_name: "k8s_clusters_count"
         column: "count"
+
+  # API usage
+  - id: APIUsageMetrics
+    source: VM
+    query: grpc_server_msg_received_total
+    transform:
+      type: JSON
+      metric: pmm_server_api_usage
+    summary: "PMM Server API usage"
+    data:
+      - metric_name: "1"
+        label: "caller_origin" # internal / external
+      - metric_name: "2"
+        label: "grpc_method"
+      - metric_name: "3"
+        label: "grpc_service"
+      - metric_name: "4"
+        label: "grpc_type"
+      - metric_name: "5" # pmm-managed / qan-api2
```

Comment:
telemetry key: pmm_server_api_usage
telemetry value:
{
  "v": [
    {
      "1": "external",
      "2": "CheckUpdates",
      "3": "server.Server",
      "4": "unary",
      "5": "pmm-managed",
      "6": 304
    },
    {
      "1": "external",
      "2": "DiscoverRDS",
      "3": "management.RDS",
      "4": "unary",
      "5": "pmm-managed",
      "6": 697
    },
    {
      "1": "external",
      "2": "GetSettings",
      "3": "server.Server",
      "4": "unary",
      "5": "pmm-managed",
      "6": 16
    },
    {
      "1": "external",
      "2": "ListArtifacts",
      "3": "backup.v1.Artifacts",
      "4": "unary",
      "5": "pmm-managed",
      "6": 788
    },
...
}

Reply:
it is not only to make it shorter, golang by default sorts JSON keys alphabetically (when we marshal map, e.g. [here](https://github.com/percona/pmm/blob/28afb77e80eecbee8de30344c075d359422cfed0/managed/services/telemetry/transform.go#L74-L74))

if we create config like 

```
    data:
      - metric_name: "a"
        label: "caller_origin" # internal / external
      - metric_name: "z"
        label: "grpc_method"
      - metric_name: "c"
        label: "grpc_service"
      - metric_name: "b"
 ````

it will create json:

```
{
 "a": ...,
 "b": ...,
 "c":....,
 "b":...
}
```

which might be problematic to parse that in telemetryd.

for example:

```
JSONExtract(
  arrayElement(arrayFilter(x -> x.1 = 'pmm_server_api_usage', pmm_server_metrics), 1).2,
                                'v',
                                'Array(Tuple(String,String,String,String,String,String))'
)
```

can be used to transform raw JSON to normal table (via materialized view)

Reply:
Sorry, i don't get. Why we are using these short metric names? Why not just use label name as metric name?

Reply:
we store these it in DB as string, it is used for:
 - minimize used space
 - make ordering obvious

Reply:
Is disk space an issue for us? So to understand what these `a`, `b`, `c` means I should find this config file, right?

Reply:
We don't need readable names here.

This will not be used in raw format, we will have materialized view that does this transformation:
![image](https://user-images.githubusercontent.com/91831719/204301183-ec27ba9a-2576-4cf2-9933-0f70b836a79c.png)

Reply:
what if we need to add some additional dimension into this tuple - shall we adjust this view?
This is exactly the issue that we wanted to get rid of by introducing Telemetry V2.

Reply:
These views create per metric, each metric can have its set of attributes. Each metric can evolve independently, if we need to introduce a breaking change we need to create new metric, otherwise it will be a maintenance hell.

If we need to change tuple—worst case we have to recreate views.